xmtp_content_types/src/transaction_reference.rs

```diff
@@ -76,11 +94,14 @@ pub struct TransactionReference {
 /// Metadata attached to the transaction reference
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct TransactionMetadata {
+    #[serde(rename = "transactionType")]
     pub transaction_type: String,
     pub currency: String,
     pub amount: f64,
     pub decimals: u32,
+    #[serde(rename = "fromAddress")]
```

Comment:
It also feels a little dangerous to manually annotate the fields that need to be transformed - is there a higher level setting we could maybe use, e.g. `#[serde(rename_all = "camelCase")]`?